### PR TITLE
fixed: test environment

### PR DESCRIPTION
**Pull request by notspiff**
fallout from https://github.com/xbmc/xbmc/pull/9389

### CODE REVIEW

**Comment by FernetMenta** [comment]
thanks!

**Comment by wsnipex** [comment]
jenkins build this please

**Comment by FernetMenta** [comment]
Travis is happy again

File: xbmc/test/TestBasicEnvironment.cpp [hunk 1]
```diff
@@ -63,6 +63,10 @@ void TestBasicEnvironment::SetUp()
   g_powerManager.Initialize();
   CSettings::GetInstance().Initialize();
 
+  g_application.m_ServiceManager.reset(new CServiceManager());
+  if (!g_application.m_ServiceManager->Init())
```
Comment (garbear) [inline xbmc/test/TestBasicEnvironment.cpp]: needs updating after #9405, yea?

Comment (FernetMenta) [inline xbmc/test/TestBasicEnvironment.cpp]: could you do this please? there may be more changes required

**Pull request merged**